Gemfile.lock.ruby-2.0

```diff
@@ -1,6 +1,5 @@
 GEM
   remote: https://rubygems.org/
-  remote: https://rubygems.org/
   remote: http://rubygems.org/
```

Comment:
Is this still needed?

Reply:
Not sure, I've just copied whats on disk right now. I'd be inclined to leave it alone for the moment and investigate with a copy of the repo....